### PR TITLE
Adjuted patterns and plain system role (bsc#1075998)

### PR DESCRIPTION
**Pull request by jsrain**

### CODE REVIEW

File: control/control.CAASP.xml [hunk 1]
```diff
@@ -275,6 +275,97 @@ textdomain="control"
...
+
+            <volumes config:type="list">
+                <!-- The / filesystem -->
+                <volume>
+                    <mount_point>/</mount_point>
+                    <!-- Default == final, since the user can't change it -->
+                    <fs_type>btrfs</fs_type>
+                    <desired_size config:type="disksize">25 GiB</desired_size>
+                    <min_size config:type="disksize">16 GiB</min_size>
+                    <max_size config:type="disksize">30 GiB</max_size>
```
Comment (jreidinger) [inline control/control.CAASP.xml:306]: is there reason for max size? There are no other mp, so unlimited should be fine not?

File: control/control.CAASP.xml [hunk 2]
```diff
@@ -275,6 +275,97 @@ textdomain="control"
...
+            </proposal>
+
+            <volumes config:type="list">
+                <!-- The / filesystem -->
+                <volume>
+                    <mount_point>/</mount_point>
+                    <!-- Default == final, since the user can't change it -->
+                    <fs_type>btrfs</fs_type>
+                    <desired_size config:type="disksize">25 GiB</desired_size>
+                    <min_size config:type="disksize">16 GiB</min_size>
```
Comment (jreidinger) [inline control/control.CAASP.xml:305]: do we have somethere reasoning for this number?

File: control/control.CAASP.xml [hunk 3]
```diff
@@ -275,6 +275,97 @@ textdomain="control"
...
+                    <max_size config:type="disksize">30 GiB</max_size>
+                    <weight config:type="integer">80</weight>
```
Comment (jreidinger) [inline control/control.CAASP.xml:307]: I think useless here as there are no other partitions.

File: control/control.CAASP.xml [hunk 4]
```diff
@@ -275,6 +275,97 @@ textdomain="control"
...
+                        <subvolume>
+                            <path>var</path>
+                            <copy_on_write config:type="boolean">false</copy_on_write>
+                        </subvolume>
+
+                        <!-- architecture specific subvolumes -->
+
+                        <subvolume>
+                            <path>boot/grub2/arm64-efi</path>
+                            <archs>aarch64</archs>
```
Comment (jreidinger) [inline control/control.CAASP.xml:344]: do we plan caasp for other arch? or it is just for safe?

File: control/control.CAASP.xml [hunk 5]
```diff
@@ -362,6 +453,13 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
         <none_role_description>
           <label>• No services started by default.</label>
         </none_role_description>
+        <micro_os_role>
+          <!-- TRANSLATORS: a label for a system role -->
+          <label>SUSE MicroOS</label>
```
Comment (jreidinger) [inline control/control.CAASP.xml:458]: new product?

File: control/control.CAASP.xml [hunk 6]
```diff
@@ -362,6 +453,13 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
...
+        </micro_os_role>
+        <micro_os_role_description>
+          <label>• No services started by default.</label>
```
Comment (jreidinger) [inline control/control.CAASP.xml:461]: I think this is not very helpful as it is identical description as MicroOS. So what is difference to Plain System role?

File: control/control.Kubic.xsl [hunk 7]
```diff
@@ -31,4 +37,11 @@
       <xsl:copy-of select="document('/CD1/control.xml')/*/n:software/n:extra_urls"/>
     </xsl:copy>
   </xsl:template>
+
+  <xsl:template match="n:micro_os_role">
+    <micro_os_role>
+      <label>openSUSE Micro OS</label>
```
Comment (jreidinger) [inline control/control.Kubic.xsl]: @sysrich any comments?

**Comment by jsrain** [comment]
The labels, as well as values will very likely need to be adjusted based on the experience (we might even add another partition later); this is the initial shot, which was reviewed by Thorsten and matches the expectation for the intial build.

File: control/control.CAASP.xml (hunk 2 above)
Comment (jsrain) [reply control/control.CAASP.xml:305]: The sizes are initial shot and no matter how we set them, we will likely adjust them later.

File: control/control.CAASP.xml (hunk 4 above)
Comment (jsrain) [reply control/control.CAASP.xml:344]: It's the same as the default proposal. Better be safe here...

**Review by jreidinger — APPROVED** [review]

**Pull request merged**